### PR TITLE
announcing the 1.14.0 release candidate

### PR DESCRIPTION
**Pull request by joemoe**

### CODE REVIEW

File: _posts/2021-08-31-release-1.14.0-rc0.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Dear Flink Community,
+
+we are excited to announce the first release candidate of Apache Flink 1.14. 🎉
+
+There are a lot of features and fixes - improving
+the unified batch and streaming experience, increased the fault tolerance by 
+reducing the in-flight data, worked on connectors,.... as well as tidied some 
+components, just to mention some - that went into this release and it wouldn't
+have been possible without your help. :)
```
Comment (infoverload) [inline _posts/2021-08-31-release-1.14.0-rc0.md]:
```suggestion
A lot of features and fixes went into this release, including improvements to the unified batch and streaming experience, an increase in fault tolerance by reducing in-flight data, and more developments on connectors and components. 

It wouldn't have been possible without your help. :)  [Around 211 people]((https://github.com/apache/flink/graphs/contributors?from=2021-03-31&to=2021-08-31&type=c) ) have made contributions!
```

File: _posts/2021-08-31-release-1.14.0-rc0.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+we are excited to announce the first release candidate of Apache Flink 1.14. 🎉
+
+There are a lot of features and fixes - improving
+the unified batch and streaming experience, increased the fault tolerance by 
+reducing the in-flight data, worked on connectors,.... as well as tidied some 
+components, just to mention some - that went into this release and it wouldn't
+have been possible without your help. :) 
+
+[Around 211 people]((https://github.com/apache/flink/graphs/contributors?from=2021-03-31&to=2021-08-31&type=c) ) have made contributions!
+Two weeks ago (August 16th) we created a feature freeze. This means that only a
```
Comment (infoverload) [inline _posts/2021-08-31-release-1.14.0-rc0.md]:
```suggestion
Two weeks ago (August 16th) we created a feature freeze. This means that only a 
```

**Review by infoverload — CHANGES_REQUESTED** [review]

File: _posts/2021-08-31-release-1.14.0-rc0.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Dear Flink Community,
+
+we are excited to announce the first release candidate of Apache Flink 1.14. 🎉
+
+A lot of features and fixes went into this release, including improvements to the 
+unified batch and streaming experience, an increase in fault tolerance by reducing 
+in-flight data, and more developments on connectors and components. 
+It wouldn't have been possible without your help. 
+[Around 211 people](https://github.com/apache/flink/graphs/contributors?from=2021-03-31&to=2021-08-31&type=c)
```
Comment (xintongsong) [inline _posts/2021-08-31-release-1.14.0-rc0.md]: I'd suggest to remove this link, as it confuses people with an inconsistent number.

**Review by xintongsong — APPROVED** [review]
Thanks @joemoe. LGTM.
I've only one minor comment. If you agree, I'll address it myself while merging this.

**Pull request closed without merging**